pkg/auth/handler/oauth/authorize.go

```diff
@@ -0,0 +1,45 @@
+package oauth
+
+import (
+	"net/http"
+
+	"github.com/gorilla/mux"
+
+	pkg "github.com/skygeario/skygear-server/pkg/auth"
+	"github.com/skygeario/skygear-server/pkg/auth/dependency/oauth/handler"
+	"github.com/skygeario/skygear-server/pkg/auth/dependency/oauth/protocol"
+)
+
+func AttachAuthorizeHandler(
+	router *mux.Router,
+	authDependency pkg.DependencyMap,
+) {
+	router.NewRoute().
+		Path("/oauth2/authorize").
+		Handler(pkg.MakeHandler(authDependency, newAuthorizeHandler)).
+		Methods("GET")
```

Comment:
Should support both GET and POST https://openid.net/specs/openid-connect-core-1_0.html#AuthRequest